modules/concepts/pages/concepts.adoc

```diff
@@ -0,0 +1,55 @@
+= Stargate concepts
+
+Stargate is a framework used to customize all aspects of data access.
+It is deployed between client applications and a database to provide an abstraction layer that can be used to shape your data access to fit your application's needs.
+
+To understand the vision of this project, see the high-level architecture diagram below.
+
+image::https://github.com/stargate/stargate/blob/master/assets/stargate-arch-high-level.png?raw=true[Stargate High-Level Architecture]
```

Comment:
What do you mean by "json" here? Is that different than "rest"?

Reply:
It refers to the document api